src/it/java/net/spy/memcached/ClientBaseCase.java

```diff
@@ -151,6 +160,29 @@ protected void restoreClusterConfig(String config, Collection<NodeEndPoint> endp
 	}
   }
 
+  protected void restoreClusterConfigForTLS(int portNumber) throws Exception {
```

Comment:
Can you simply update `restoreClusterConfig()` method to handle TLS instead of adding this new method?